### PR TITLE
Add opentracing stubs

### PR DESCRIPTION
**Pull request by kasium**
Hey guys,

I would like to add stubs for the opentracing project (https://github.com/opentracing/opentracing-python). I already asked if they can consider typing but never received an answer.

This change just contains a few stubs since I would like to first lay a foundation and keep commits small.

### CODE REVIEW

**Comment by srittau** [comment]
> I would like to add stubs for the opentracing project (https://github.com/opentracing/opentracing-python). I already asked if they can consider typing but never received an answer.

Thanks!

> This change just contains a few stubs since I would like to first lay a foundation and keep commits small.

This is fine, but did you consider using stubgen to generate a first version of the stubs? This can save a lot of work.

File: stubs/opentracing/opentracing/tags.pyi [hunk 1]
```diff
@@ -0,0 +1,22 @@
+SPAN_KIND: str = ...
```
Comment (srittau) [inline stubs/opentracing/opentracing/tags.pyi:1]: Nit: Leave of ` = ...` here.

**Comment by Akuli** [comment]
To temporarily silence pylint warnings, you can exclude opentracing in `pyrightconfig.stricter.json`.

**Comment by kasium** [comment]
> To temporarily silence pylint warnings, you can exclude opentracing in `pyrightconfig.stricter.json`.

Thanks, but I guess I just need to add some further Any's. They are also a good hint what is missing

**Comment by Akuli** [comment]
>add some further Any's. They are also a good hint what is missing

For argument and return types, we prefer no type hint at all over `Any`.

**Comment by kasium** [comment]
> > add some further Any's. They are also a good hint what is missing
> 
> For argument and return types, we prefer no type hint at all over `Any` in this situation.

Thanks. I removed the any where possible and set the config

**Review by srittau — APPROVED** [review]
A cursory examination looks good. Thanks!

**Pull request merged**